kinks/level_4/introduction_to_programming_i/README.md

```diff
@@ -123,15 +129,50 @@ Note that the `distance`/`km`/`miles_travelled` property is increasing towards t
 
 Once implemented correctly, remember the logic, as you will likely need to create similar functions things in the stages ahead.
 
-#### 702, stage 4
+#### 702, Stage 4
 _"My suspect's car didn't have enough space to pull out in front of me before speeding off, so the chase wouldn't work as expected. I deliberately got myself suspended so I could grab another case, and it worked fine."_ (reported by [@dannycallaghan](https://github.com/dannycallaghan))
 
-#### 802, stage 2
+#### 801, All stages
+More spotlight stages, which may cause Firefox users to not see the spotlights. 
+To get it to work, comment out `blendMode(DARKEST)` during your work and to uncomment again before submission. Or don't.
+
+Tips:
+- The 1st seat in the 1st row will be `seats[0][0]` 
```

Comment:
Extra space at the end.

Reply:
Since those bullets points are full sentences, a period would work well instead.